membership-attribute-service/app/services/PaymentFailureAlerter.scala

```diff
@@ -52,6 +51,14 @@ object PaymentFailureAlerter extends LoggingWithLogstashFields {
         case None => Future.successful(None)
       }
 
+      def getProductDescription(subscription: Subscription[SubscriptionPlan.AnyPlan]) = if (subscription.asMembership.isDefined) {
+        s"${subscription.plan.productName} membership"
+      } else if (subscription.asContribution.isDefined) {
+        "contribution"
+      } else { //TODO SEE WHAT GENERIC TEXT WE SHOULD RETURN HERE
```

Comment:
Do we need to exclude the alertText in the /me/mma- endpoints for the products we do not include in "alertAvailableFor" in /me ?

Reply:
I don't think so. We deliberately released the change for the the alertText for /me/mma-membership first since the info is still useful even if they didn't get to the manage page via the banner. Also, we need to make a change for each tab à la https://github.com/guardian/frontend/compare/aw-digipack-warn to see the alertText anyway.